src/org/zaproxy/zap/view/widgets/WritableFileChooser.java

```diff
@@ -26,6 +28,21 @@ public WritableFileChooser(File currentDirectory) {
 	public void approveSelection() {
 		File selectedFile = getSelectedFile();
 
+        File checkFile = selectedFile;
```

Comment:
This and following line should use tabs for indentation (same chars as surrounding code).